cime_config/acme/allactive/testlist_allactive.xml

```diff
@@ -1,289 +1,5 @@
 <?xml version="1.0"?>
 <testlist>
-  <compset name="A">
```

Comment:
No - it's because A and X tests are now defined in driver_cpl/cime_config/testdefs/testlist_drv.xml